gym/wrappers/rescale_action.py

```diff
@@ -0,0 +1,28 @@
+import numpy as np
+
+import gym
+from gym import spaces
+
+
+class RescaleAction(gym.ActionWrapper):
+    r"""Rescale the continuous action space of the environment from a range of [a, b]. """
+    def __init__(self, env, a, b):
+        assert isinstance(env.action_space, spaces.Box), 'expected Box action space.'
+        assert np.less_equal(a, b).all()
+        super(RescaleAction, self).__init__(env)
+        if np.isscalar(a):
```

Comment:
You could squeeze these 4 lines into just 1 line and allow broadcasting with something like:
```suggestion
        self.a = np.zeros(env.action_space.shape, dtype=env.action_space.dtype) + a
```